components/audit_table/audit_table.jsx

```diff
@@ -330,11 +325,14 @@ AuditTable.propTypes = {
     showUserId: PropTypes.bool,
     showIp: PropTypes.bool,
     showSession: PropTypes.bool,
+    currentUser: PropTypes.object.isRequired,
+    getUser: PropTypes.func.isRequired,
+    getByName: PropTypes.func.isRequired,
 };
 
 export default injectIntl(AuditTable);
 
-export function formatAuditInfo(audit, formatMessage) {
+export function formatAuditInfo(audit, formatMessage, currentUser, getByName, getUser) {
```

Comment:
I think we need to start breaking this component like per if condition below,:
- new `AuditChannelInfo` for `actionURL.indexOf('/channels') === 0`
- new `AuditOAuthInfo` `actionURL.indexOf('/oauth') === 0`
- new `AuditUserInfo` for `actionURL.indexOf('/users') === 0`
- new `AuditTimestampInfo` for `audit.create_at`